packages/ra-data-simple-prisma/src/audit/auditHandler.ts

```diff
@@ -1,9 +1,15 @@
 import { prisma } from "@prisma/client";
-import { AuditActions, AuditOptions } from "./types";
+import { AuditOptions, defaultAuditOptions } from "./types";
 import { Request } from "../Http";
 
-export const auditHandler = async (options: AuditOptions, request: Request) => {
+export const auditHandler = async (
+  options: AuditOptions,
+  request: Request,
+  created?: any
+) => {
   const action = request.body.method.split(/(?=[A-Z])/)[0];
+  options = { ...options, ...defaultAuditOptions };
```

Comment:
```suggestion
  options = { ...defaultAuditOptions, ...options, };
```
So defaults can be overridden